### PR TITLE
prepare v0.10 release

### LINKED ISSUE #337: Actix 0.10 roadmap
Rough thoughts:

- [x] #324 
  - [x] update `trust-dns-resolver` to 0.19.0 or higher (#345)
- [x] #335
- [x] #405 make `Request` `Send`

(deferred to later release) #314 (https://github.com/actix/actix/issues/337#issuecomment-652185348)

### PR DESCRIPTION
**Pull request by robjtede**
## PR Type
Release / Docs

## PR Checklist
- [x] Documentation comments have been added / updated.
- [x] A changelog entry has been made for the appropriate packages.
- [x] Format code with the latest stable rustfmt

## Overview
closes #337, #370

### CODE REVIEW

File: .github/workflows/macos.yml [hunk 1]
```diff
@@ -34,19 +34,21 @@ jobs:
...
-            ~/.cargo/git
-            ~/.cargo/bin
-          key: ${{ matrix.version }}-x86_64-apple-darwin-cargo-${{ hashFiles('**/Cargo.lock') }}
-      - name: Cache cargo build
-        uses: actions/cache@v2
-        with:
-          path: target
-          key: ${{ matrix.version }}-x86_64-apple-darwin-cargo-build-trimmed-${{ hashFiles('**/Cargo.lock') }}
+
+      # - name: Cache cargo dirs
```
Comment (cetra3) [inline .github/workflows/macos.yml:38]: Is there a reason that these are commented out?

File: LICENSE-APACHE [hunk 2]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2017-NOW Nikolay Kim
+   Copyright 2017-NOW Actix Team
```
Comment (cetra3) [inline LICENSE-APACHE:189]: Should probably read `Nikolay Kim + Actix Team` unless Nikolay has given the OK for this?

**Review by cetra3 — COMMENTED** [review]
Looks good, a few nitpicks

File: LICENSE-APACHE (hunk 2 above)
Comment (robjtede) [reply LICENSE-APACHE:189]: I've messaged+emailed him, waiting on a reply. If no response comes back soon I'll revert it.

File: .github/workflows/macos.yml (hunk 1 above)
Comment (robjtede) [reply .github/workflows/macos.yml:38]: Yeah the macos build was thowing nonsense build errors and figured it was cache artifacts related.

**Review by fafhrd91 — APPROVED** [review]

File: LICENSE-APACHE (hunk 2 above)
Comment (fafhrd91) [reply LICENSE-APACHE:189]: actix team is totally fine, i missed it.

**Review by JohnTitor — APPROVED** [review]

File: .github/workflows/macos.yml (hunk 1 above)
Comment (popzxc) [reply .github/workflows/macos.yml:38]: I think then it makes sense to add a comment why these lines were commented. Commented code without an explanation usually raises questions.

**Pull request merged**